templates/bundles/PrestaShopBundle/Admin/Sell/Order/Order/Blocks/Create/cart.html.twig

```diff
@@ -152,9 +152,11 @@
 {% endset %}
 
 <script id="js-product-custom-text-template" type="text/template">
-      {{ requiredFieldMarkTemplate }}
-      <label for="" class="js-product-custom-input-label"></label>
-      <input name="" type="text" class="form-control js-product-custom-input">
+  <div>
```

Comment:
```suggestion
  <div class="product-custom">
```

Could we already add a class there? So it's easier to select

Reply:
@NeOMakinG :heavy_check_mark: Thanks :smile_cat: